src/main/java/Task.java

```diff
@@ -0,0 +1,40 @@
+public class Task {
+
+    protected String description;
+    protected boolean isTaskDone;
+
+    public Task(String description){
+        this.description = description;
+        this.isTaskDone = false;
+    }
+
+    public Task(String description, boolean isTaskDone){
+        this.description = description;
+        this.isTaskDone = isTaskDone;
+    }
+
+    public String getDescription(){
```

Comment:
I like how you follow the Java coding standards and stuck with K&R brackets, as well as writing your methods in camelCase